source/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,163 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+You can query against geospatial indexes in several ways with
+`geospatial query operators
+<https://docs.mongodb.com/manual/geospatial-queries/index.html>`_.
```

Comment:
Can we focus this on the idea of geospatial data rather than geospatial indexes? The indexes are obviously very important but I don't think they appropriately motivate the page as part of the introduction.

Reply:
Agreed, reworded to introduce the concept better. Added a second sentence about required indexes though, since that's an important prerequisite.